src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -183,17 +183,17 @@ export interface RestorationFilingIF {
     legalType: CorpTypeCd
   }
   restoration: {
-    applicationDate: string // YYYY-MM-DD
-    approvalType: ApprovalTypes
+    applicationDate?: string // YYYY-MM-DD
+    approvalType?: ApprovalTypes
     contactPoint: ContactPointIF
-    courtOrder: CourtOrderIF
-    expiry: string // YYYY-MM-DD
-    nameRequest: NameRequestFilingIF
-    nameTranslations: NameTranslationIF[]
-    noticeDate: string // YYYY-MM-DD
+    courtOrder?: CourtOrderIF
+    expiry?: string // YYYY-MM-DD
+    nameRequest?: NameRequestFilingIF
+    nameTranslations?: NameTranslationIF[]
+    noticeDate?: string // YYYY-MM-DD
     offices: RegisteredRecordsAddressesIF | object
     parties: PartyIF[]
-    relationships: RelationshipTypes[]
+    relationships?: RelationshipTypes[]
```

Comment:
NB: just because the schema says these are optional (and I think the schema needs to be updated) doesn't mean the UI should treat these as optional.